libs/sdk-ui-kit/src/Form/InputWithNumberFormat.tsx

```diff
@@ -0,0 +1,191 @@
+// (C) 2007-2020 GoodData Corporation
+import React from "react";
+import memoize from "lodash/memoize";
+import { InputPure, InputPureProps } from "./InputPure";
+
+import { Separators } from "./typings";
+
+// Highest number (BIGINT) according to gooddata documentation help.gooddata.com object-datatypes
+export const MAX_NUMBER = 10 ** 15;
+
+// Max number of digits right to decimal point according to gooddata documentation help.gooddata.com object-datatypes
+export const MAX_DECIMAL_POINT_NUMBERS = 6;
```

Comment:
This export seems unnecessary, this is only used ins this file.

Reply:
fixed